extensions/ccui/layouts/UILayoutWebGLRenderCmd.js

```diff
@@ -233,7 +261,7 @@
 
     proto.scissorClippingVisit = function(parentCmd){
         cc.renderer.pushRenderCommand(this._beforeVisitCmdScissor);
-        cc.ProtectedNode.prototype.visit.call(this._node, parentCmd);
+        ccui.ProtectedNode.WebGLRenderCmd.prototype.visit.call(this, parentCmd);
```

Comment:
I think this should be the line which fixed the issue, so what do other modifications do ?